### DISCUSSION TITLE
How to use built-in variables in my local variables?

### DISCUSSION THREAD

**Discussion by pingsoli** [xmake-io/xmake#1474] [Q&A] [answered]
### 描述讨论详情
```
target("test")
    local my_test_path = "../$(plat)/$(mode)/"
    print(my_test_path)
```
the print:
```
../$(plat)/$(mode)/
```

even I run `xmake f -m debug`, nothing changed.

**Comment by waruqi** [comment, marked as answer]
use option, https://xmake.io/#/manual/configuration_option

and do not use `print()` in description scope. https://xmake.io/#/guide/syntax_description
